.github/auto_assign.yml

```diff
@@ -0,0 +1,27 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: true
+
+# A list of reviewers to be added to pull requests (GitHub user name)
+reviewers:
+  - myunggyuns
+  - sunnieeeYoon
+  - changchanghwang
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 2
```

Comment:
Why is it 2 instead of 0?

Reply:
Even if the number of contributors increases, only 2 people review.